Daily Acceptance Test failed: golangci-lint installation

golangci-lintのインストールでエラー

```
  GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
  # github.com/golangci/golangci-lint/pkg/golinters
  /go/src/github.com/golangci/golangci-lint/pkg/golinters/golint.go:21:14: l.LintPkg undefined (type *"github.com/golangci/lint-1".Linter has no field or method LintPkg)
  /go/src/github.com/golangci/golangci-lint/pkg/golinters/gomnd.go:13:6: cannot use magic_numbers.Analyzer (type *"github.com/tommy-muehle/go-mnd/vendor/golang.org/x/tools/go/analysis".Analyzer) as type *"golang.org/x/tools/go/analysis".Analyzer in array or slice literal
  make: *** [GNUmakefile:41: tools] Error 2
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 ## 2.3.1 (Unreleased)
 
+MISC
+
+* Fix broken CI - install golangci-lint via install script [GH-735] (@yamamoto-febc)
+
 ## 2.3.0 (2020-04-20)
 
 * Startup Script Parameters [GH-731] (@yamamoto-febc)

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -38,7 +38,7 @@ tools:
 	GO111MODULE=off go get github.com/bflad/tfproviderdocs
 	GO111MODULE=off go get github.com/bflad/tfproviderlint/cmd/tfproviderlintx
 	GO111MODULE=off go get github.com/client9/misspell/cmd/misspell
-	GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $$(go env GOPATH)/bin v1.24.0
 
 
 .PHONY: build-envs

